core/bignum/divmod_spec.rb

```diff
@@ -14,10 +14,13 @@
     @bignum.divmod(4).should == [2305843009213693965, 3]
     @bignum.divmod(13).should == [709490156681136604, 11]
 
-    @bignum.divmod(4.0).should == [2305843009213693952, 0.0]
-    @bignum.divmod(13.0).should == [709490156681136640, 8.0]
+    not_supported_on :opal do
+      @bignum.divmod(4.0).should == [2305843009213693952, 0.0]
+      @bignum.divmod(13.0).should == [709490156681136640, 8.0]
+
+      @bignum.divmod(2.0).should == [4611686018427387904, 0.0]
```

Comment:
Changing 4.0 to 4.5 would be a workaround for that. but i do not know if you would like to change this.